README.md

```diff
@@ -1 +1,22 @@
-# termionus
\ No newline at end of file
+# termionus
+
+We are will eventually create a logo but for now you can look at this >\_
+
+## What is termionus?
+
+termionus is an open source terminal emulator for all operating systems.
```

Comment:
I think "all operating systems" might be too over-promising.

Let's mention for now that it can be run on Windows and Linux.

```suggestion
termionus is an open source terminal emulator for Windows and Linux operating systems.
```